### ISSUE TITLE
[Suggestion] Show update news and current running version in self-updater even if there are no updates

### ISSUE THREAD

**Issue by barfin** [ValveSoftware/steam-for-linux#8649]
#### Your system information

* Steam client version (build number or date): Jun 7 2022
* Opted into Steam client beta?: No

#### Please describe your issue in as much detail as 
1. Show update news so we don't have to do a google search to see the changelog after updating
2. and show the current version so we don't have go to another section to see which changes are going to apply after looking at the changelog

basically something like this
![ssssss](https://user-images.githubusercontent.com/37030489/175539845-95da0f0a-468b-4023-9752-02e2fb705061.png)
